build.gradle

```diff
@@ -50,6 +50,7 @@ dependencies {
 	compile 'commons-codec:commons-codec:1.10'
 	compile 'commons-httpclient:commons-httpclient:3.1'
 	compile 'commons-logging:commons-logging:1.2'
+	compile 'commons-io:commons-io:2.4'
```

Comment:
Agreed.
663699  removes the jars in lib that are also defined as gradle dependencies.

It's interesting it also did not take very long at all for the lib jars and the gradle versions to drift, already some were no longer the same version.